Getting-Started/Setup/Server-Setup/Load-Balancing/flexible-advanced.md

```diff
@@ -48,9 +49,10 @@ The first thing to do is create a couple classes for your front-end servers and
 			return null;
 		}
 	}
+```
 
 then you'll need to swap the default `DatabaseServerRegistrar` for the your custom registrars during application startup.
-You'll need to create an [ApplicationEventHandler](/Documentation/Reference/Events/Application-Startup) and override `ApplicationStarting`. 
+You'll need to create an [ApplicationEventHandler](../../../../Reference/Events/Application-Startup.md) and override `ApplicationStarting`. 
```

Comment:
```suggestion
You'll need to create an [ApplicationEventHandler](../../../../Reference/Events/Application-Startup) and override `ApplicationStarting`. 
```

So when the .md is added to a URL it will find the file, but it will also find it without the .md extension - the bonus is if we some day extend upon that article and move it to a folder with an image folder and call it index.md, then it will find that folder name instead and then automatically look for the index file 😉

Reply:
hmm, if we don't have the .md the docs are not navigable on GitHub though which I believe we are trying to maintain as well as via Our?

Reply:
I would argue having it maintainable on Our would be more important than linking between pages on Github, but we can talk about it on our Curators meeting tomorrow 🙂